applications/StructuralMechanicsApplication/tests/test_dynamic_schemes.py

```diff
@@ -46,22 +46,25 @@ def _base_spring_test_pseudo_static_scheme(self, current_model, scheme_name = "p
         current_disp = 0.0
         current_vel = init_velocity
         # Solve the problem
-        while(time <= end_time):
+        while time <= end_time:
             time = time + dt
             step = step + 1
             mp.CloneTimeStep(time)
             mp.ProcessInfo[KratosMultiphysics.STEP] = step
 
             self.strategy.Solve()
 
-            current_analytical_displacement_y = current_disp + current_vel * dt - current_vel * beta * mass * dt/2.0
+            current_analytical_displacement_y = current_disp + current_vel * dt * (1.0 - 5.0 * beta * mass)
```

Comment:
why do you change the analytic solution? does it work with the previous one?

Reply:
The previous one was not correct; i updated the predict and update, that's why it failed